Windows cross-compilation not work because mupdf HAVE_OBJCOPY

Build from linux for target windows causes:

```
# cargo build --release --target x86_64-pc-windows-gnu

target/x86_64-pc-windows-gnu/release/deps/libmupdf_sys-006e76309b72ff78.rlib(hyphen.o):hyphen.c:(.rdata$.refptr._binary_resources_hyphen_hyph_all_zip_end[.refptr._binary_resources_hyphen_hyph_all_zip_end]+0x0): undefined reference to `_binary_resources_hyphen_hyph_all_zip_end'
```

It works only with fix:

```diff
main.rs:200
-        if target.arch == "wasm32" {
+        if target.arch == "wasm32" || target.os == "windows" {
            self.make_bool("HAVE_OBJCOPY", false);
        }
```

--- a/mupdf-sys/make.rs
+++ b/mupdf-sys/make.rs
@@ -196,9 +196,11 @@ impl Make {
         self.make_bool("HAVE_GLUT", false);
         self.make_bool("HAVE_CURL", false);
 
-        if target.arch == "wasm32" {
-            self.make_bool("HAVE_OBJCOPY", false);
-        }
+        // let cc-rs handle compiler selection, not the Makefile.
+        // fill to not be empty and therefore `uname` being used.
+        self.make_var("OS", "not empty");
+
+        self.make_bool("HAVE_OBJCOPY", target.os == "linux");
 
         self.make_bool("verbose", true);
 

